src/DIRAC/WorkloadManagementSystem/JobWrapper/Watchdog.py

```diff
@@ -964,7 +964,10 @@ def getNodeInformation(self):
         """Retrieves all static system information"""
         result = {}
         result["HostName"] = socket.gethostname()
-        result["CPU(MHz)"] = psutil.cpu_freq()[0]
+        try:
+            result["CPU(MHz)"] = psutil.cpu_freq()[0]
+        except Exception:
+            result["CPU(MHz)"] = 0
```

Comment:
That's not ideal -- it will become a job parameter, entered in the DB (MySQL, or ES). 

Any better way of doing it?

Reply:
What is it trying to do?

Naively I think the field doesn't make any sense to look at these days with dynamic frequency scaling and heterogeneous architectures like big.LITTLE/Alder Lake/DynamIQ. I'd propose removing both the `CPU(MHz)` and `CacheSize(kB)` fields and let people use `ModelName` instead if they want to use this information.

Reply:
You have a point here. Go ahead.

Reply:
I'll do it in a separate PR as I'll need to look check carefully where `CPU(MHz)` is used. Depending on how invasive it is I'll pick either the 8.0 or 8.1 release.